lib/wallet/src/Cardano/Wallet.hs

```diff
@@ -3845,8 +3848,8 @@ instance HasSeverityAnnotation WalletLog where
 
 data TxSubmitLog
     = MsgSubmitTx BuiltTx (BracketLog' (Either ErrSubmitTx ()))
-    | MsgSubmitExternalTx (Hash "Tx") (BracketLog' (Either ErrPostTx Tx))
-    | MsgRetryPostTx (Hash "Tx") (BracketLog' (Either ErrPostTx ()))
+    | MsgSubmitExternalTx TxId (BracketLog' (Either ErrPostTx SealedTx))
```

Comment:
Hi @HeinrichApfelmus 

Just curious, what's the rationale for logging a `SealedTx` instead of a `Tx` here?
(Asking as I couldn't find an obvious reason in the comments.)

Thanks!

Reply:
Understood, thanks for the explanation!